<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>12</y>
     <width>611</width>
     <height>211</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="cityName">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;имя города&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>442</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>541</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="var1">
      <property name="text">
       <string>Вариант ответа 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="var2">
      <property name="text">
       <string>Вариант ответа 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="var3">
      <property name="text">
       <string>Вариант ответа 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="var4">
      <property name="text">
       <string>Вариант ответа 4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="quest">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>262</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
